src/components/footer/Footer.jsx

```diff
@@ -26,30 +26,30 @@ export default function Footer() {
             <h4>Categories</h4>
             <ul className="list-style-none">
               <li>
-                <Link to="/products" element={<ProductListing />}>Weights</Link>
+                <Link to="/products">Weights</Link>
```

Comment:
We can also use ```NavLink``` right?
Is there a reason you chose ```Link``` over ```NavLink```

Reply:
`NavLink` is used when you want add styles to an active route link. Here the links only serve the purpose of navigating, I don't want to style them. Hence `Link` is used here